ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResourceTest.java

```diff
@@ -257,51 +263,64 @@ public void shouldReturnServiceUnavailableIfTimeoutWaitingForCommandSequenceNumb
   }
 
   @Test
-  public void shouldNotCreateAdminClientForPullQuery() throws Exception {
-    // When:
-    testResource.streamQuery(
-        serviceContext,
-        new KsqlRequest(PULL_QUERY_STRING, Collections.emptyMap(), null)
-    );
+  public void shouldNotCreateExternalClientsForPullQuery() {
+    // Given
+    testResource.configure(new KsqlConfig(ImmutableMap.of(
+        StreamsConfig.APPLICATION_SERVER_CONFIG, "something:1",
+        KsqlConfig.KSQL_PULL_QUERIES_SKIP_ACCESS_VALIDATOR_CONFIG, true
+    )));
 
-    // Then:
+    // Expect
     verify(serviceContext, never()).getAdminClient();
-  }
+    verify(serviceContext, never()).getConnectClient();
+    verify(serviceContext, never()).getSchemaRegistryClient();
+    verify(serviceContext, never()).getTopicClient();
```

Comment:
test bug: The `verify` calls need to come _after_ the test case.  This is verifying the calls are not being made by the test setup.